xmipp

```diff
@@ -894,18 +894,23 @@ if __name__ == '__main__':
     elif mode=="compile":
         Nproc = 8 if n<3 else sys.argv[2]
         if n<=3:
-            ok = compile(Nproc)
+            ok, strConfigInfo = compile(Nproc)
             module = 'Xmipp'
         else:
+            strConfigInfo = buildConfig.ensure_version()
             compileDependencies(Nproc)
             ok = compileModule(Nproc,sys.argv[3])
             module = sys.argv[3]
         if ok:
+            if strConfigInfo:
+                print(yellow(strConfigInfo))
```

Comment:
other build systems (e.g. CMake) print the configuration at the beginning. Like that, you would also avoid the code repetitions (see like 905-906, 912-913 ...

Reply:
But the idea is to print it at the end of the process ("compile", "compileAndInstall" or "All"). 
To avoid code repetition, i 've created a function on utils.py